FLEDGE_k_anonymity_server.md

```diff
@@ -178,21 +178,40 @@ of network and computational overhead, and multi-party PIR, which has less
 overhead but the additional complexity of operating two non-colluding servers
 with consistent copies of the dataset.
 
-#### Anonymous tokens to replace low entropy identifiers
-
-We're working on researching and testing a privacy improvement to low-entropy
-browser identifiers.  The $j$-bit identifier, `b`, is constant for a given
-browser, which allows some inferences to be made by the `Join` server, in
-spite of collisions between users.  To improve the privacy of this scheme
-and increase the accuracy of our cardinality calculations, while maintaining
-our ability to prevent abusive traffic, we're researching additions to Trust
-Token APIs.
-
-We are working on extending anonymous tokens in a way that enables users to
-obtain signatures on the set being joined without revealing the set to `Sign`.
-We also aim to enable additional functionality that prevents users from
-obtaining multiple tokens for the same set, letting us verify on the server
-that browsers aren't joining the same set more frequently than they should.
+#### Anonymous Counting Tokens to replace low-entropy identifiers
+
+We're working on researching and testing a privacy improvement to
+[low-entropy](#low-entropy-identifiers) browser identifiers.  The $j$-bit
+identifier, `b`, is constant for a given browser, which allows some inferences
+to be made by the `Join` server, in spite of collisions between users.
+To improve the privacy of this scheme and increase the accuracy of our
+cardinality calculations, while maintaining our ability to prevent abusive
+traffic, we're developing a new token scheme that we're calling _Anonymous
+Counting Tokens_.
+
+Anonymous Counting Tokens will allow a token issuer (the `Sign` server)
+to issue tokens to a client that are associated with a value that the
+client provides.  The issuer will be able to ensure that the client
+can obtain only a single token for a given value without knowing
+the value the client is requesting a token for, i.e. the tokens are
+[blind](https://en.wikipedia.org/wiki/Blinding_(cryptography)).
+
+The `Sign` server will use Anonymous Counting Tokens to issue tokens signed
+for a single set hash, in contrast to the low-entropy identifier design where
+tokens are signed for a value derived from a first party identity the `Sign`
+server is given by the client.  Each user will be able to request only a
+single token for each set hash and the `Join` server will verify, in what
+will become a `Join(t, s)` call, that the token was issued for the set hash
+`s` that the client is attempting to join.
```

Comment:
I think this needs another sentence to bring home where the magic is hiding!  "It may be surprising that this is even possible without the token issuer learning anything about the set hashes joined by a particular user, but <cryptographic technique X> does indeed allow this outcome."

Reply:
Done :)  I think you might know this, but there's a conference paper in the works on Anonymous Counting Tokens, so we'll have more details to share publicly after that's published.

Reply:
Thanks, a reference to the upcoming paper looks good for now.  Please remember to come back and change this into a link once there is an actual publication!